_posts/2020-01-24-quarkus-wins-devies-award.adoc

```diff
@@ -2,7 +2,7 @@
 layout: post
 title: Quarkus wins DEVIES award
 date: 2020-01-24
-tags: quarkus awards
+tags: announcement awards
```

Comment:
are we expecting 50 blongs on awards and wanting to filter them out?

Reply:
well, when I did this there was discussions on what we do when we have more than one devies award; so I treated it as something interesting to have a page for like `user-story`